cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -228,7 +228,7 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
             }
 
         // Find a list of candidate symbols. In most cases, we can just perform a lookup by name
-        // which either performs an unqualified lookup beginning from the current scope "up-wards",
+        // which either performs an unqualified lookup beginning from the current scope "upwards",
```

Comment:
Can we make the SymbolResolver an EOG starter pass then? We should probably also move the TU/Import ordering logic to the `Pass` class then.

Reply:
I'm trying to implement this in #2188 and #2187 but there's an issue in the go integration tests which I don't understand yet.

Reply:
The issue with the SymbolResolver is fixed now